lib/_tls_common.js

```diff
@@ -175,6 +176,52 @@ exports.createSecureContext = function createSecureContext(options, context) {
     }
   }
 
+  if (options.pskCallback && c.context.enablePskCallback) {
+    if (typeof options.pskCallback !== 'function') {
+      throw new ERR_INVALID_ARG_TYPE('pskCallback',
+                                     'function', options.pskCallback);
+    }
+
+    c.context.onpskexchange = (identity, maxPskLen, maxIdentLen) => {
```

Comment:
It'd be a little nicer to turn this into a top-level function.  Compare `onhandshakestart()` in `lib/_tls_wrap.js`.